product_secondary_unit/models/product_secondary_unit_mixin.py

```diff
@@ -0,0 +1,115 @@
+# Copyright 2021 Tecnativa - Sergio Teruel
```

Comment:
wrong copyright ?

Reply:
No this part was be added in v 13. I got it from there  https://github.com/OCA/product-attribute/blob/13.0/product_secondary_unit/models/product_secondary_unit_mixin.py